### ISSUE TITLE
Error to compile on Ubuntu 14.4

### ISSUE THREAD

**Issue by brust** [deadtrickster/prometheus.erl#52]
It's not able to compile `prometheus.erl` on **Ubuntu 14.4**

`mix.exs`:
```
defp deps do[{:prometheus, "~> 3.1.0"}] end
```

```
$ mix local.rebar
$ rm -rf deps
$ rm -rf _build

$ mix deps.get
Running dependency resolution...
Dependency resolution completed:
  prometheus 3.1.1
* Getting prometheus (Hex package)
  Checking package (https://repo.hex.pm/tarballs/prometheus-3.1.1.tar)
  Fetched package

$ mix compile
===> Compiling prometheus
===> Unable to run pre hooks for 'compile', command 'compile' in namespace 'protobuf' not found.
** (Mix) Could not compile dependency :prometheus, "/home/ubuntu/.mix/rebar3 bare compile --paths "/home/application/current/_build/prod/lib/*/ebin"" command failed. You can recompile this dependency with "mix deps.compile prometheus", update it with "mix deps.update prometheus" or clean it with "mix deps.clean prometheus"
```

```
$ cat /etc/*release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.3 LTS"
NAME="Ubuntu"
VERSION="14.04.3 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.3 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
```

```
$ elixir --version
Erlang/OTP 19 [erts-8.2] [source] [64-bit] [smp:4:4] [async-threads:10] [kernel-poll:false]

Elixir 1.4.0
```

**Comment by deadtrickster** [comment]
Hi, thanks for the report

I wonder whether it's 14.04 (or whatever) specific. 
You can try [prometheus.ex](https://hex.pm/packages/prometheus_ex) or this patch: https://github.com/deadtrickster/prometheus.erl/pull/51/commits/9d8d942aeaf09f732bc6d964819ce27dccc9e05f in the meantime.

**Comment by deadtrickster** [comment]
Please let me know if the patch worked for you

**Comment by brust** [comment]
@deadtrickster Thanks for your help! 😄 

It's works!

```
{:prometheus, "~> 3.1.0", git: "https://github.com/deadtrickster/prometheus.erl.git", override: true},```

**Comment by deadtrickster** [comment]
Glad to hear it works for you somehow :-)

So it's a problem with 3.1.1... 


If you have time could you please do this:

- while using git switch to HEAD commit and compile
- switch back to hex but fix version from hex to 3.1.0 and compile

Please let me know the results. Thanks!

**Comment by brust** [comment]
Fixed on `3.2.2`

**Comment by deadtrickster** [comment]
cc @vernomcrp
